room/compiler-processing/src/main/java/androidx/room/compiler/processing/ksp/KspElement.kt

```diff
@@ -61,4 +61,10 @@ internal abstract class KspElement(
             KSFileAsOriginatingElement(it)
         }
     }
+
+    override val docComment: String? by lazy {
+        // Not yet implemented in KSP.
```

Comment:
can you add a TODO to the beginning of this line.

also, thanks for linking the issue, i have a small bash script that parses ksp links so that we know what we are waiting for from KSP.